sample/src/main/res/layout/item_horizontal_list.xml

```diff
@@ -14,10 +14,16 @@
   ~ limitations under the License.
   -->
 
-<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/post_list"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingLeft="@dimen/normal_space"
-    android:clipToPadding="false"/>
+    android:descendantFocusability="blocksDescendants">
```

Comment:
Looks like you mean, use the `LinearLayout` to grab/block the focus, to remove the focus of sub-RecyclerView.